nginx.conf

```diff
@@ -0,0 +1,38 @@
+worker_processes 1;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    upstream likes_service {
+        server host.docker.internal:8080;
+    }
+
+    upstream users_service {
+        server host.docker.internal:8081;
+    }
+
+    upstream posts_service {
+        server host.docker.internal:8082;
+    }
+
+    server {
+        listen 80;
+
+        # Route for Likes
+        location /likes {
+            proxy_pass http://likes_service;
```

Comment:
shouldn't we proxy to the needed endpoint?

Reply:
If each service will expose their api with service name prefix then there is no need to map address nginx will do it byself